### PR TITLE
OpenThread Example Improvement

### LINKED ISSUE #10295: [Feature] Display in the console the network identifier to which the card is connected, in addition to the role
### Related area

OpenThread

### Hardware specification

ESP32-C6, ESP32-H2

### Is your feature request related to a problem?

Hello,

I'm using this example: [RouterNode.ino](https://github.com/espressif/arduino-esp32/blob/master/libraries/OpenThread/examples/SimpleThreadNetwork/RouterNode/RouterNode.ino) to test the connection to my existing Thread network.

I want to make sure that my board is connected to the right network (PANID). 
Is there any way of accessing this information and displaying it in the console?

Regards

### Describe the solution you'd like

Display in the console the network identifier to which the card is connected, in addition to the role.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### I have checked existing list of Feature requests and the Contribution Guide

- [X] I confirm I have checked existing list of Feature requests and Contribution Guide.

### PR DESCRIPTION
**Pull request by SuGlider**
## Description of Change

Adds a new example that mixes OpenThread native API with different Arduino OThreadCLI API calls.
New example is `ExtendedRouterNode.ino`.

Adds OpenThread Native API calls to the current Router and Leader Examples.

Fixes the Library `keywords.txt` file that didn't use TAB to separate fields.

## Tests scenarios
Tested using two ESP32-C6 one for the Leader and the other for the Child/Router.
Used the modified examples.

## Related links
Closes #10295 
Related to #9486

### CODE REVIEW

File: libraries/OpenThread/examples/SimpleThreadNetwork/LeaderNode/LeaderNode.ino [hunk 1]
```diff
@@ -31,7 +35,53 @@ void setup() {
...
+    uint16_t panId = otLinkGetPanId(aInstance);
+    Serial.printf("PanID: 0x%04x\r\n", panId);
+    // Extended PAN ID
+    const otExtendedPanId *extPanId = otThreadGetExtendedPanId(aInstance);
+    Serial.printf("Extended PAN ID: ");
+    for (int i = 0; i < OT_EXT_PAN_ID_SIZE; i++) {
+      Serial.printf("%02x", extPanId->m8[i]);
+    }
+    Serial.println();
+    // Nettwork Key
```
Comment (P-R-O-C-H-Y) [inline libraries/OpenThread/examples/SimpleThreadNetwork/LeaderNode/LeaderNode.ino:61]: "Network" typo
```suggestion
    // Network Key
```

File: libraries/OpenThread/examples/SimpleThreadNetwork/RouterNode/RouterNode.ino [hunk 2]
```diff
@@ -31,7 +35,53 @@ void setup() {
...
+    uint16_t panId = otLinkGetPanId(aInstance);
+    Serial.printf("PanID: 0x%04x\r\n", panId);
+    // Extended PAN ID
+    const otExtendedPanId *extPanId = otThreadGetExtendedPanId(aInstance);
+    Serial.printf("Extended PAN ID: ");
+    for (int i = 0; i < OT_EXT_PAN_ID_SIZE; i++) {
+      Serial.printf("%02x", extPanId->m8[i]);
+    }
+    Serial.println();
+    // Nettwork Key
```
Comment (P-R-O-C-H-Y) [inline libraries/OpenThread/examples/SimpleThreadNetwork/RouterNode/RouterNode.ino:61]: same here :)
```suggestion
    // Network Key
```

**Review by P-R-O-C-H-Y — APPROVED** [review]
Just a typo to fix :) esle LGTM

**Comment by SuGlider** [comment]
Typos are automatically fixed by CI.

**Review by me-no-dev — APPROVED** [review]

**Pull request merged**